statsmodels/genmod/cov_struct.py

```diff
@@ -938,6 +951,14 @@ class Equivalence(CovStruct):
     using this covariance structure, missing data should be removed
     before constructing the model.
 
+    If using `labels`, after a model is defined using the covariance
+    structure it is possible to remove a label pair from the second
+    level of the `pairs` dictionary to force the corresponding
+    covariance to be zero.
```

Comment:
I think there are some further convenience functions that can be added in a future PR.  One would be to remove an equivalence class so the parameter is estimated as zero.  Another would be to merge two equivalence classes so that the parameters become equated to each other.